sig-auth/governance.md

```diff
@@ -0,0 +1,153 @@
+    by a majority of subproject contributors (those having some role in the subproject).
+  - *MAY* select additional subproject owners through a [super-majority] vote amongst subproject
+    owners.  This *SHOULD* be supported by a majority of subproject contributors (through
+    [lazy-consensus] with fallback on voting).
+  - *SHOULD* work to ensure technical excellence within area
+    - actively work to reduce complexity of the area
+    - define general acceptance criteria for work as needed (code, test, documentation guidelines,
+      etc)
+    - consider cross-project interactions in technical decisions
+    - prioritize urgent issues appropriately (e.g. addressing critical bugs, security
+      vulnerabilities, or test failures)
+  - Number: 1-3 per subproject
+  - Defined in [sigs.yaml] [OWNERS] files
+
+- Members
+  - *MUST* maintain health of at least one subproject or the health of the SIG
+  - *MUST* show sustained contributions to at least one subproject or to the SIG
+  - *SHOULD* hold some documented role or responsibility in the SIG and / or at least one subproject
+    (e.g. reviewer, approver, etc)
+  - *MAY* build new functionality for subprojects
+  - *MAY* participate in decision making for the subprojects they hold roles in
+  - *SHOULD* include all reviewers and approvers in [OWNERS] files for subprojects
+  - *MUST* remain active in the role and are automatically removed from the position if they are
+    unresponsive for > 1 year.
+  - *MAY* be removed if not proactively working with other Subproject Owners to fulfill
+    responsibilities.
+
+## Organizational management
+
+- SIG meets bi-weekly on [zoom] with [agenda in meeting notes]
+  - *SHOULD* be facilitated by chairs unless delegated to specific Members
+- SIG overview and deep-dive sessions organized for Kubecon
+  - *SHOULD* be organized by chairs unless delegated to specific Members
+
+- Contributing instructions defined in the SIG CONTRIBUTING.md
+
+### Project management
+
+#### Subproject creation
+
+- Subprojects *MAY* be created by [KEP] proposal and *MUST* be approved by at least 1 tech lead, and
+  accepted by [lazy-consensus] with fallback on majority vote of SIG Technical Leads.  The result
+  *SHOULD* be supported by the majority of SIG members.
+  - KEP *MUST* establish subproject owners
+  - [sigs.yaml] *MUST* be updated to include subproject information and [OWNERS] files with
+    subproject owners
+  - Where subprojects processes differ from the SIG, they must document how
+    - e.g. if subprojects release separately - they must document how release and planning is
+      performed
+
+### Technical processes
+
+Subprojects of the SIG *MUST* use the following processes unless explicitly following alternatives
+they have defined.
+
+- Proposing and making decisions
+  - Proposals *SHOULD* be sent as [KEP] PRs and published to [kubernetes-sig-auth] as an
```

Comment:
It would be great to document here who needs to be set as an approver when doing a KEP.  My guess is that for stuff that impacts most of the SIG it'd be a TL (with acceptance from that TL and lazy consensus around that choice).  Similarly for something that is solely scoped to a subproject it'd be a subproject owner w/ lazy consensus.